core/templates/domain/feedback_message/ThreadMessage.model.ts

```diff
@@ -26,8 +26,10 @@ export interface ThreadMessageBackendDict {
   'entity_id': string;
   'message_id': number;
   'text': string;
-  'updated_status': string;
-  'updated_subject': string;
+  // Status and Subject for frontend instances of thread message
+  // domain objects should be null initially.
```

Comment:
This seems suspect. This is a backend dict so it represents something passed to the backend. Is it allowed to pass null to the backend? If so, explain its meaning.

Otherwise, better to find a way where the fields are populated fully when constructing the dict. If you can't do that, explain why and we can discuss it further.

Reply:
I am actually not very sure of the property usage here, we are sending null to the backend see line 42,43 . I found this PR https://github.com/oppia/oppia/pull/8658 for reference where this file was actually created but couldn't find anything related to it.Can you please shed some light on the usage here?

Reply:
You will need to look into the backend code to see the reason. Hint: Look here https://github.com/oppia/oppia/blob/68de4ec14bc130de6056fe0cd3b2bc0322cc7ce1/core/controllers/feedback.py#L77

Reply:
@vojtechjelinek Thanks for directing, found this `feedback_services.py`
![image](https://user-images.githubusercontent.com/60519303/121143241-79bd6800-c85a-11eb-93d9-d3bdd32364f0.png)
These properties should exist only when the status or subject changes and hence non-existent if not.So I think this can be `null`.